### PR TITLE
Updated deprecated links in build-custom-mobile-apps.rst

### PR DESCRIPTION
**Pull request by Rutam21**
#### Summary

Updated deprecated links in build-custom-mobile-apps.rst

#### Ticket Link

NA

### CODE REVIEW

File: source/deploy/build-custom-mobile-apps.rst [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Building and Distributing Your Own Custom Mattermost Mobile Apps
 ================================================================
 
-You can build and distribute custom versions of the Mattermost Mobile App. Choosing this approach means `you've decided not to use the mobile app Mattermost has made available <https://docs.mattermost.com/mobile/use-prebuilt-mobile-apps.html>`__ through public app stores. This also means that you've decided to `host your own push proxy service <https://docs.mattermost.com/mobile/mobile-hpns.html#mobile-push-notifications>`__.
+You can build and distribute custom versions of the Mattermost Mobile App. Choosing this approach means `you've decided not to use the mobile app Mattermost has made available <https://docs.mattermost.com/deploy/use-prebuilt-mobile-apps.html>`__ through public app stores. This also means that you've decided to `host your own push proxy service <https://docs.mattermost.com/deploy/build-custom-mobile-apps.html>`__.
```
Comment (cwarnermm) [inline source/deploy/build-custom-mobile-apps.rst:4]:
```suggestion
You can build and distribute custom versions of the Mattermost Mobile App. Choosing this approach means `you've decided not to use the mobile app Mattermost has made available <https://docs.mattermost.com/deploy/use-prebuilt-mobile-apps.html>`__ through public app stores. This also means that you've decided to `host your own push proxy service <https://docs.mattermost.com/deploy/mobile-hpns.html#host-your-own-push-proxy-service>`__.
```

Comment (cwarnermm) [reply source/deploy/build-custom-mobile-apps.rst:4]: I've provided an updated URL.

File: source/deploy/build-custom-mobile-apps.rst [hunk 2]
```diff
@@ -21,7 +21,7 @@ When you decide to build your own Mattermost Mobile apps, you have multiple ways
 Setting Up Push Notifications
 -----------------------------
 
-See our `Mobile Push Notifications <https://docs.mattermost.com/mobile/mobile-hpns.html>`__ documentation to learn about setting up push notifications for custom built mobile apps.
+See our `Mobile Push Notifications <https://docs.mattermost.com/deploy/build-custom-mobile-apps.html#setting-up-push-notifications>`__ documentation to learn about setting up push notifications for custom built mobile apps.
```
Comment (cwarnermm) [inline source/deploy/build-custom-mobile-apps.rst:24]:
```suggestion
See our `Mobile Push Notifications <https://docs.mattermost.com/deploy/mobile-hpns.html>`__ documentation to learn about setting up push notifications for custom built mobile apps.
```

Comment (cwarnermm) [reply source/deploy/build-custom-mobile-apps.rst:24]: I've provided an updated URL.

**Review by cwarnermm — COMMENTED** [review]
Thank you for this docs contribution! I've provided updated URL links inline.

**Comment by Rutam21** [comment]
@cwarnermm The suggested changes have been applied. Please review and merge.

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**